app/src/main/java/org/mifos/selfserviceapp/ui/activities/LoginActivity.java

```diff
@@ -59,7 +59,7 @@ protected void onCreate(Bundle savedInstanceState) {
     @Override
     public void onLoginSuccess(String userName) {
         loginStatus = true;
-        showToast(getString(R.string.toast_welcome, userName));
+        showToast(getString(R.string.toast_welcome, userName.replaceFirst("\\s++$", "")));
```

Comment:
Revert this change, this is not making any sense, our Back-end API is good to handle these things.